src/index.js

```diff
@@ -209,9 +218,9 @@ function trackAdded(participant, track) {
     const color = colorHash.hex(track.id);
     track.on('message', data => {
       const { mouseDown, mouseCoordinates: { x, y } } = JSON.parse(data);
-      if (mouseDown) {
-        drawCircle(canvas, color, x, y);
-      }
+
+      // TODO: Improve this by using mouseDown to draw a path on a canvas
+      drawCircle(canvas, color, x * window.innerWidth, y * window.innerHeight);
```

Comment:
It seems like we would always draw mouse coordinates with this change, instead of when the mouse is held down. I think we should retain the previous behavior. On mobile, where "clicking" isn't so much a thing, could you just send position with `mouseDown` set to `true` when the user's finger is on the screen?